micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmThreadMetrics.java

```diff
@@ -66,6 +66,11 @@ public void bindTo(MeterRegistry registry) {
                 .description("The current number of live threads including both daemon and non-daemon threads")
                 .baseUnit(BaseUnits.THREADS).register(registry);
 
+        Gauge.builder("jvm.threads.started", threadBean, ThreadMXBean::getTotalStartedThreadCount).tags(tags)
```

Comment:
Is there a daemon/live getter so that this can be consistent to `jvm.threads.daemon`/`jvm.threads.live`?

Reply:
Not sure what this means. Can you explain a bit more on this?

Reply:
There are two types of threads in Java: daemon and non-daemon. See the previous few lines: `jvm.threads.daemon` and `jvm.threads.live`, see: https://docs.oracle.com/javase/8/docs/api/java/lang/Thread.html

Reply:
Yeah. I am aware of that. But those OS implementations give us the current number of threads active. But this is more about the rate at which threads are getting created. And unfortunately, the native implementation doesn't differentiate between daemon and non-daemon for this method.